src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj

```diff
@@ -1,24 +1,30 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
   <PropertyGroup>
-	<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
-	<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;net6.0-ios;net6.0-maccatalyst;net6.0-macos</TargetFrameworks>
-	<TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">net6.0-windows10.0.18362;netstandard2.0;net6.0-android</TargetFrameworks>
+    <!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
+    <TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;net6.0-ios;net6.0-maccatalyst;net6.0-macos</TargetFrameworks>
+    <TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">net6.0-ios;net6.0-windows10.0.18362;netstandard2.0;net6.0-android</TargetFrameworks>
```

Comment:
I don't think I understand this. Why are some TargetFrameworks in both conditions?

Reply:
Removed the condition and combined the TargetFrameworks